components/PlayerCard/index.js

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import styled from "styled-components";
+export default function PlayerCard({ player }) {
+  return (
+    <StyledCard>
+      <Image src="/img/players/darts-player.jpg" alt={player.name} width="150" height="150" />
```

Comment:
```suggestion
      <Image src="/img/players/darts-player.jpg" alt={player.name} width={150} height={150} />
```

next image want those values to come as numbers :)